### PR TITLE
[DOCFIX]cn for Getting-Started

### PR DESCRIPTION
**Pull request by TeddyBear1314**

### CODE REVIEW

File: docs/cn/Getting-Started.md [hunk 1]
```diff
@@ -1,46 +1,314 @@
...
 layout: global
-title: 开始
+title: 快速上手指南
 group: User Guide
 priority: 0
 ---
 
-* Table of Contents
-{:toc}
+试用Alluxio最简单的方式是单机本地安装。在这个快速上手指南里，我们会在你的本地机器上安装Alluxio，挂载样本数据，对Alluxio中的数据执行一些基本操作。具体来说，包括:
```
Comment (RongGu) [inline docs/cn/Getting-Started.md]: '试用'->'体验'；‘我们会在你的本地机器上安装Alluxio‘ ->‘我们会引导你在本地机器上安装Alluxio’

**Comment by RongGu** [comment]
@TeddyBear1314 Thanks for the contributation. I have some minor comments. Please handle that, thanks!

**Comment by RongGu** [comment]
LGTM.

**Comment by apc999** [comment]
LGTM

**Pull request merged**